contracts/base/Lenders.sol

```diff
@@ -118,6 +119,10 @@ contract Lenders is LendersInterface {
         external
         isLendingPool()
         isValid(recipient)
+        isInitialized()
+        whenNotPaused()
+        whenLendingPoolNotPaused()
```

Comment:
This function is marked as `isLendingPool`. So if the lending pool is paused, it cannot be called from anywhere. Therefore it doesnt need the `whenLendingPoolNotPaused`, as the lending pool will deal with the pausing

Reply:
I don't believe youve made this change?

Reply:
It's weird. The change was pushed. We will review it after merging.